index.js

```diff
@@ -111,6 +115,24 @@ module.exports = async (input = 'patch', options) => {
 		]);
 	}
 
+	if (runStatusChecks) {
+		tasks.add([{
+			title: 'Ensure GitHub status checks have passed',
```

Comment:
`Ensure` => `Ensuring` to fit with the other titles.

Reply:
Changed in 05bc3d33cf97585ab2a4a11e138072f1253e85f9.